### ISSUE TITLE
Editorial: Missing definition words in Understanding Guideline 1.3: Adaptable

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#2791]
In the intent section, in WCAG 2.0, the bullets read:
- **structure:** the way the parts of a Web page are organized in relation to each other; and the way a collection of Web pages is organized
- **presentation:** rendering of the content in a form that can be perceived by users

In WCAG 2.1 and 2.2 the words "structure" and "presentation" are missing, so the bullets read: 
- the way the parts of a Web page are organized in relation to each other; and the way a collection of Web pages is organized
- rendering of the content in a form that can be perceived by users
